development/docker-compose/readme.md

```diff
@@ -30,16 +30,16 @@ For the setup, you must have the following:
 
 ### Initial set up
 
-1. Install Colima to run Docker and work with Docker Compose:
+1. Install your preferred Docker engine to work with Docker Compose, in this documentation we'll use [Colima](https://github.com/abiosoft/colima):
     ```shell
     brew install colima docker docker-compose docker-credential-helper
     colima start --cpu 4 --memory 16
     ```
-   Workaround to fix Colima issue [#764](https://github.com/abiosoft/colima/issues/764) in order to build the Identity Auth Server image using docker compose:
+   If you are using Apple Silicon (Macbook M1, M2, M3, etc), we recommend enabling Rosetta for best compatibility - Requires v0.5.3 and MacOS >= 13 (Ventura):
    ```shell
-    docker buildx create --driver-opt 'image=moby/buildkit:rootless' 
+    colima start --cpu 4 --memory 16 --vm-type=vz --vz-rosetta --mount-type virtiofs
```

Comment:
According to my tests only with Rosetta enabled we can run SQL Server x86 emulated images in a M1 laptop, for example